src/ClientStream.php

```diff
@@ -61,6 +71,16 @@ public function __construct(// @phpstan-ignore-line
      */
     public function write($request)
     {
+        if ($this->logger && $request instanceof Message) {
```

Comment:
add a comment to explain this (or log the string `$request`)
```suggestion
        // In some cases, $request can be a string
        if ($this->logger && $request instanceof Message) {
```